.github/workflows/deploy-ccv0-demo.yaml

```diff
@@ -0,0 +1,131 @@
+on:
+  issue_comment:
+    types: [created, edited]
+
+name: deploy-ccv0-demo
+
+jobs:
+  check-comment-and-membership:
+    runs-on: ubuntu-latest
+    if: |
+      github.event.issue.pull_request
+      && github.event_name == 'issue_comment'
+      && github.event.action == 'created'
+      && startsWith(github.event.comment.body, '/test_kata_deploy')
```

Comment:
@Jakob-Naucke, this is the comment that will trigger the action.  You'll need to change that to `/deploy-ccv0-demo` (or whatever you want to call).